cmd/docker-app/inspect.go

```diff
@@ -3,24 +3,37 @@ package main
 import (
 	"github.com/docker/app/internal/inspect"
 	"github.com/docker/app/internal/packager"
+	"github.com/docker/app/types"
 	"github.com/docker/cli/cli"
 	"github.com/docker/cli/cli/command"
+	cliopts "github.com/docker/cli/opts"
 	"github.com/spf13/cobra"
 )
 
+var (
+	inspectSettingsFile []string
+	inspectEnv          []string
+)
+
 // inspectCmd represents the inspect command
 func inspectCmd(dockerCli command.Cli) *cobra.Command {
-	return &cobra.Command{
-		Use:   "inspect [<app-name>]",
-		Short: "Shows metadata and settings for a given application",
+	cmd := &cobra.Command{
+		Use:   "inspect [<app-name>] [-s key=value...] [-f settings-file...]",
+		Short: "Shows metadata, settings and summary of compose file for a given application",
```

Comment:
Grammar nit:
I think this should be "Shows metadata, settings and a summary of compose file for a given application" (added "a" before summary) unless you are compressing for line length.

I'm wavering towards saying it should be "summary of the compose file" too (with the "the" in it), although not 100% sure on that one.

Reply:
Thank you! I definitely need some grammar nits :)